### PR TITLE
Removed Color::gray

### PR DESCRIPTION
**Pull request by hbina**
It was marked for removal in Godot 3.1

### CODE REVIEW

**Review by aaronfranke — APPROVED** [review]
Probably best if this is on the 4.0 milestone.

**Comment by hbina** [comment]
Should I also remove any mention of this from the documentation?

**Comment by Calinou** [comment]
@hbina Yes, the associated documentation should be removed as the method no longer exists.

**Comment by hbina** [comment]
I have removed a mention of Color::gray() in the documentation...there's only 1. Not sure if I missed any other.

File: modules/gdnative/gdnative_api.json [hunk 1]
```diff
@@ -507,13 +507,6 @@
           ["const godot_color *", "p_self"]
         ]
       },
-      {
-        "name": "godot_color_gray",
-        "return_type": "godot_real",
-        "arguments": [
-          ["const godot_color *", "p_self"]
-        ]
-      },
```
Comment (bojidar-bg) [inline modules/gdnative/gdnative_api.json]: This would break compatibility with older versions of GDNative libraries, not sure if we want to do that.

Comment (aaronfranke) [reply modules/gdnative/gdnative_api.json]: Unless we want to leave a method that does nothing, it's probably necessary. Breaking ABI compatibility should be fine for 4.0.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**